### ISSUE TITLE
Remove favourites limit or introduce a like system

### ISSUE THREAD

**Issue by smileyhead** [ppy/osu-web#5510]
So, recently I've bumped into the 100 limit for favourited beatmaps, and I think that limit should be removed. I don't really see the point of limiting them in the first place. I frequently use my favourites list to quickly be able to download the same maps on multiple systems.
Or, if you really need them to be limited, introduce another system (likes, for example) that is unlimited, but also has a page to list all the maps you've liked.

**Comment by peppy** [comment]
As a temporary solution, if you are a supporter this increases to 1,000

<img width="317" alt="image" src="https://user-images.githubusercontent.com/191335/72663096-c455fe80-3a31-11ea-8794-30c8b1fd7cd9.png">

Going forward we will likely introduce a more flexible system, but this relies on lazer.

**Comment by revelsix** [comment]
yes yes yes 100x yes to a like/dislike system. would be helpful for the Popular Beatmaps section and (given proper lazer integration) would encourage more people to rate beatmaps
